compose-local-deps.yml

```diff
@@ -29,18 +29,18 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
-  cs-elasticsearch:
-    network_mode: bridge
-    container_name: cs-elasticsearch
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.0
-    environment:
-      - cluster.name=cs-isaac
-      - "network.host=0.0.0.0"
-      - node.name=localhost
-      - cluster.initial_master_nodes=localhost
-    ports:
-      - "9201:9200"
-      - "9301:9300"
+#  cs-elasticsearch:
+#    network_mode: bridge
+#    container_name: cs-elasticsearch
+#    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.0
+#    environment:
+#      - cluster.name=cs-isaac
+#      - "network.host=0.0.0.0"
+#      - node.name=localhost
+#      - cluster.initial_master_nodes=localhost
+#    ports:
+#      - "9201:9200"
+#      - "9301:9300"
```

Comment:
@Scratchcat1 - would you mind adding another commit to this branch reverting this change I'm sure you didn't intend to commit? Thanks!